tests/test_objective_funs.py

```diff
@@ -867,12 +867,13 @@ def test(coil, grid=None):
     def test_coil_min_distance(self):
         """Tests minimum distance between coils in a coilset."""
 
-        def test(coils, mindist, grid=None):
+        def test(coils, mindist, grid=None, test_method=False, expect_intersect=False):
```

Comment:
what is `test_method`?

Reply:
it was before when I was testing the method's warning when things intersect. I removed now